applications/ThermalTGV.cc

```diff
@@ -260,10 +260,6 @@ void ThermalTGV<dim>::solve(const unsigned int &level)
   velocity->reinit();
   error.reinit(temperature->solution);
   initialize();
-
-  // Advances the time to t^{k-1}, either t^0 or t^1
-  for (unsigned int k = 1; k < time_stepping.get_order(); ++k)
-    time_stepping.advance_time();
```

Comment:
@sebglane  The removal of these lines is what causes the convergence rate to change in this branch. If added again one gets:
```
                               Temperature convergence table
==============================================================================================
level    dt    cells dofs    hmax           L2                 H1               Linfty       
    8 1.00e-01 65536 66049 5.52e-03 3.422045e-04     - 1.459549e-02     - 6.387538e-04     - 
    8 5.00e-02 65536 66049 5.52e-03 9.723780e-05 -1.82 1.431300e-02 -0.03 1.483619e-04 -2.11 
    8 2.50e-02 65536 66049 5.52e-03 3.221458e-05 -1.59 1.429360e-02 -0.00 1.721641e-05 -3.11 
    8 1.25e-02 65536 66049 5.52e-03 1.626306e-05 -0.99 1.429249e-02 -0.00 1.647540e-05 -0.06 
    8 6.25e-03 65536 66049 5.52e-03 1.262168e-05 -0.37 1.429246e-02 -0.00 2.492815e-05  0.60 
    8 3.12e-03 65536 66049 5.52e-03 1.178615e-05 -0.10 1.429247e-02  0.00 2.696698e-05  0.11 
    8 1.56e-03 65536 66049 5.52e-03 1.157736e-05 -0.03 1.429247e-02  0.00 2.749647e-05  0.03 
    8 7.81e-04 65536 66049 5.52e-03 1.154823e-05 -0.00 1.429247e-02  0.00 2.754702e-05  0.00 
```
i.e. the results from `mit_benchmark`. One can then assume that the error in `master` is not caused by the `boundary_conditions.*` files.